packages/config/config/devices/0x0086/zw141.json

```diff
@@ -0,0 +1,219 @@
+// Aeotec ZW141
+// Nano Shutter
+{
+	"manufacturer": "Aeotec",
+	"manufacturerId": "0x0086",
+	"label": "ZW141",
+	"description": "Nano Shutter",
+	"devices": [
+		{
+			"productType": "0x0303",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3132
+		},
+		{
+			"productType": "0x0903",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3269
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"35": {
+			"label": "Moving Time From Up (left) to Down (right) For Curtain",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 127,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"80": {
+			"label": "Notification to Send to Association Group 1 on State Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send nothing",
+					"value": 0
+				},
+				{
+					"label": "Send Basic CC report",
+					"value": 1
+				}
+			]
+		},
+		"85": {
+			"label": "Operation Mode of External Switch",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "S1 for closing, S2 for opening",
+					"value": 0
+				},
+				{
+					"label": "S1 and S2 perform same action",
+					"value": 1
+				}
+			]
+		},
+		"120": {
+			"label": "External Switch Mode For S1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
+					"value": 0
+				},
+				{
+					"label": "3-way switch mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary push button mode",
+					"value": 3
+				},
+				{
+					"label": "Enter automatic identification mode",
+					"value": 4
+				}
+			]
+		},
+		"121": {
+			"label": "External Switch Mode For S2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
```

Comment:
Have you tried to find in the manual what this is for? If you can't, then remove it and adjust the min value.

Reply:
The manual doesn't explain in any detail what it is, and I can't find any other resources explaining it further. Should I go ahead and remove it? This is from the manual:

<img width="775" alt="Screen Shot 2021-02-11 at 8 54 03 PM" src="https://user-images.githubusercontent.com/1899920/107721821-98945f00-6cab-11eb-9e5e-945f76784d55.png">